README.md

```diff
@@ -0,0 +1,136 @@
+# Componentz
+
+![npm](https://img.shields.io/npm/v/@euk-labs/componentz)
+![NPM](https://img.shields.io/npm/l/@euk-labs/componentz)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/Eureka-Shoulders/componentz/CI)
+![npm](https://img.shields.io/npm/dw/@euk-labs/componentz)
+
+Componentz is an MUI-based UI library that can provide simple and performative ways to use components.
+
+The goal is not to replace the MUI, but to be an extension
+
+## Installation
+
+MUI is the base of this library so it is the first installation
+
+```bash
+# Using npm:
+npm install @mui/material @emotion/react @emotion/styled
+
+# Using yarn:
+yarn add @mui/material @emotion/react @emotion/styled
+```
+
+For dependency injection we will use the inversify lib
```

Comment:
Also provide the Inversify Docs link for users know more about what are we doing